packages/@aws-cdk/aws-cloudfront/lib/experimental/edge-function.ts

```diff
@@ -137,6 +137,9 @@ export class EdgeFunction extends Resource implements lambda.IVersion {
   public configureAsyncInvoke(options: lambda.EventInvokeConfigOptions): void {
     return this.lambda.configureAsyncInvoke(options);
   }
+  public addFunctionUrl(options?: lambda.FunctionUrlOptions): lambda.FunctionUrl {
```

Comment:
Maybe this method should return the `url` as a `string`?

Reply:
It is common to return the object that was just created. I would leave it as-is.